engine/src/test/java/io/zeebe/engine/processor/workflow/variable/mapping/ActivityOutputMappingTest.java

```diff
@@ -72,6 +72,16 @@
         scopeVariables(tuple("z", "{\"y\":2}"))
       },
       {"{'x': {'y': 2}}", mapping(b -> b.zeebeOutput("x.y", "z")), scopeVariables(tuple("z", "2"))},
+      {
+        "{'z': {'x': 1}, 'y': 2}",
+        mapping(b -> b.zeebeOutput("y", "z.y")),
```

Comment:
This is the only one which failed without the fix :sweat_smile:

Reply:
Correct. It would be more work to write a similar test for message and job. However, the other case (two variables into one document) was not tested yet. So, I think it is good to have one ;)